src/Libraries/CoreNodeModels/Input/ColorPalette.cs

```diff
@@ -116,7 +146,7 @@ public override IEnumerable<AssociativeNode> BuildOutputAst(List<AssociativeNode
         /// <returns></returns>
         public override string ToString()
         {
-            return string.Format("Color(Alpha = {3}, Red = {0}, Green = {1}, Blue = {2})", dsColor.Alpha, dsColor.Red, dsColor.Green, dsColor.Blue);
+            return string.Format("Color(Alpha = {3}, Red = {0}, Green = {1}, Blue = {2})", DsColor.Alpha, DsColor.Red, DsColor.Green, DsColor.Blue);
```

Comment:
this looks wrong to me - seems these are in the wrong order, the numbers should be 0,1,2,3.